client/js/_author/components/common/editor_upload_modal.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import Modal from 'react-modal';
+
+const tagNameMap = {
+  audio: 'Audio',
+  img: 'Image',
+  video: 'Video',
+};
+
+export default function EditorUploadModal(props) {
+  function insertMedia() {
+    if (!props.mediaUrl) return;
```

Comment:
This just says if we don't have a url, don't put <img src="undefined" /> inside of tinymce